09-mapping.Rmd

```diff
@@ -277,9 +297,11 @@ many_palette_plotter(c("PRGn", "YlGn", "Set2"), 7,
 <!-- how to reverse palettes with - -->
 <!-- references -->
 
+<!-- This is quite an abrupt change of topics and does not connect well with the previous paragraphs on palette options. -->
 The **tmap** package offers many methods for specifying how variables are plotted on the map, in addition to setting `breaks` manually as we have already seen in this subsection.
 They are set with the `style` argument and some of the most useful methods are illustrated in Figure \@ref(fig:break-styles).
 
+<!-- Is there a way to improve the legends, so far they are not really appealaing -->
 ```{r break-styles, fig.cap="Illustration of different binning methods set using the style argument in tmap.", echo=FALSE}
```

Comment:
Is there a way to improve the legends, so far they are not really appealing.

Reply:
Please take a look at the updated figure here http://geocompr.robinlovelace.net/adv-map.html#fig:break-styles  following this commit which I think fixes this issue: https://github.com/Robinlovelace/geocompr/commit/b3aeda6ecc6be741591c1ef507c8d0ebe894ac92